### PR TITLE
editor: enable reference field

### PR DESCRIPTION
**Pull request by drjova**

### CODE REVIEW

File: record-editor/src/app/shared/config/hep/core.config.ts [hunk 1]
```diff
@@ -376,16 +379,19 @@ export const coreHep: JsonEditorConfig = {
             conference_record: {
               refFieldConfig: {
                 anchorBuilder: anchorBuilder,
+                displayInputField: true,
               },
             },
             journal_record: {
               refFieldConfig: {
                 anchorBuilder: anchorBuilder,
+                displayInputField: true,
```
Comment (harunurhan) [inline record-editor/src/app/shared/config/hep/core.config.ts:388]: `journal_title` autocomplete this I think

File: record-editor/src/app/shared/config/hep/core.config.ts [hunk 2]
```diff
@@ -187,6 +187,7 @@ export const coreHep: JsonEditorConfig = {
             record: {
               refFieldConfig: {
                 anchorBuilder: anchorBuilder,
+                displayInputField: true,
```
Comment (harunurhan) [inline record-editor/src/app/shared/config/hep/core.config.ts:190]: `legacy_name` autocompletion is populating this.

**Review by harunurhan — COMMENTED** [review]
how about other actively edited collections?

(authors?, conferences, institutions)

**Comment by drjova** [comment]
@harunurhan updated

**Review by harunurhan — APPROVED** [review]

**Pull request merged**